src/components/Dropdown/DropdownOption.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC, CSSProperties, ReactNode, useEffect } from 'react';
+
+export interface IDropdownOptionProps {
+  /** children must be React Element */
+  children?: ReactNode;
+  /** set customized css class */
+  className?: string;
+  /** set customized css style */
+  cssStyle?: CSSProperties;
+  /** callback provided by Dropdown */
+  setSelected?: (val: string, children: ReactNode) => void;
+  /** value for this option */
+  value?: string;
+  /** is the active / default option */
+  active?: boolean;
+}
+
+const DropdownOption: FC<IDropdownOptionProps> = (props) => {
+  const { className, children, cssStyle, setSelected, value, active } = props;
+
+  let classNames = 'dropdown__option';
+  if (className) {
+    classNames = ['dropdown__option', className].join(' ');
+  }
+
+  const passToDropdown = () => {
+    if (setSelected) {
+      // call the callback function provided by parent
+      const selectedValue = value ? value : '';
+      setSelected(selectedValue, children);
+    }
+  };
+
+  useEffect(() => {
+    // check if active is set when mounted
+    if (active) {
+      passToDropdown();
```

Comment:
the active logic will trigger the onChange Event at the start.  The real-world situation is the user prefers to have a default selected option without trigger the onChange Event.

Reply:
Ok, I will change the logic here